src/image.rs

```diff
@@ -0,0 +1,146 @@
+use super::{ImVec2, ImVec4, Ui};
+use std::marker::PhantomData;
+use std::collections::HashMap;
+use std::os::raw::c_void;
+use sys;
+
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Ord, PartialOrd, Hash)]
```

Comment:
What's the semantic meaning of Ord here? I know it's easy to derive since the content is just usize, but is there any meaningful reason to order texture IDs? If not, let's just drop Ord and PartialOrd

Reply:
It's essentially "which texture was added first". It's not *that* useful, but outside of my UI I use ordering by texture IDs as a cheap hack to collate objects with the same texture adjacent in the draw list, to reduce draw calls.

Reply:
Sequential ordering is a property of the generator, not the IDs themselves, so Ord seems slightly out of place. But I suppose there's no harm since IDs are newtype numbers anyway.